### PR TITLE
refactor(changelog): Simplify release notes functions

### PR DESCRIPTION
**Pull request by zharinov**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/workers/repository/update/pr/changelog/release-notes.ts [hunk 1]
```diff
@@ -419,36 +419,32 @@ export async function addReleaseNotes(
...
-      await packageCache.set(
-        cacheNamespace,
-        cacheKey,
-        releaseNotes,
-        cacheMinutes
-      );
+    releaseNotes ??= await getReleaseNotesMd(input.project, v);
+    releaseNotes ??= await getReleaseNotes(input.project, v, config);
+
+    // Small hack to force display of release notes when there is a compare url
```
Comment (zharinov) [inline lib/workers/repository/update/pr/changelog/release-notes.ts:436]: @rarkins Do we still need this? If so, what this hack exactly does?

Comment (rarkins) [reply lib/workers/repository/update/pr/changelog/release-notes.ts:440]: No idea, please check the git history for hints

Comment (zharinov) [reply lib/workers/repository/update/pr/changelog/release-notes.ts:440]: Checked, it was 2017, still have no idea. Will try to run with and without it.

Comment (zharinov) [reply lib/workers/repository/update/pr/changelog/release-notes.ts:440]: Anyway, I think we could merge this PR without taking in account this aspect

**Review by rarkins — APPROVED** [review]

**Pull request merged**